tooling/helium-server/worker/ssr.js

```diff
@@ -46,14 +46,30 @@ async function handleSsr(url, authToken = null, userAndAppearanceToken = null) {
     sha256,
     authToken
   );
-  const { httpResponse } = pageContext;
+
+  const { httpResponse, redirectTo } = pageContext;
+
+  let { statusCode, body } = httpResponse;
```

Comment:
So we don't want to try to destructure `httpResponse` here since the check on line 70 tells us `httpResponse` could be something other than an object. Think we'd be fine to leave the destructuring of `httpResponse` where it was, and just hardcode the `302` into the Response.